yaksh/models.py

```diff
@@ -278,6 +287,29 @@ def get_test_case(self, **kwargs):
 
         return test_case
 
+    def _add_and_get_files(self, zip_file):
+        files = FileUpload.objects.filter(question=self)
+        for file in files:
```

Comment:
A little confusing since you later go on to use file.file.path etc. I would recommend 'for f in files' or 'for fi in files'